psi4/driver/driver_cbs.py

```diff
@@ -343,23 +354,16 @@ def scf_xtpl_truhlar_2(functionname: str, zLO: int, valueLO: float, zHI: int, va
 
     Notes
     -----
-    The extrapolation is calculated according to [2]_:
-    :math:`E_{total}^X = E_{total}^{\infty} + \beta X^{-\alpha}, \alpha = 3.4`
-
-    References
-    ----------
-
-    .. [2] Truhlar, Chem. Phys. Lett. 294 (1998) 45-48,
```

Comment:
We should keep the references _somewhere_, so users have an idea what values to choose.

Reply:
Yes, they're kept in the "named function aliases" section, where the references actually make sense.